README.md

```diff
@@ -290,6 +290,35 @@ jobs:
 ```
 </details>
 
+### TestIO
+
+<details>
+
+<summary>This GitHub Action can be used to trigger a test on the external crowd-testing platform TestIO from a pull request.</summary>
+
+```yml
+name: TestIO - Trigger test from PR
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  trigger-testio-test:
+    uses: Staffbase/gha-workflows/.github/workflows/template_testio_trigger_test.yml@v2.3.0
+    with:
+      # optional: the slug you received from TestIO, defaults to 'staffbase'
+      testio-slug: your TestIO slug
+      # required: ID of the product on the TestIO platform to which the triggered test should be assigned to
```

Comment:
I would make it consistent and we only add the optional information for the other workflows. 
```suggestion
      # ID of the product on the TestIO platform to which the triggered test should be assigned to
```

Reply:
Please also for the other comments. 😆